public/app/plugins/datasource/elasticsearch/specs/elastic_response.test.ts

```diff
@@ -1469,3 +1489,41 @@ describe('ElasticResponse', () => {
     });
   });
 });
+
+interface MockedElasticResponse {
```

Comment:
@gwdawson I just wanted to double check these, to see if we really need them. Can't we use existing interfaces outside of this test file?

Reply:
just double checked and it seems there isn't already a pre-existing interface to fit this.

Reply:
Thank you for checking!